front/src/actions/map.js

```diff
@@ -35,6 +36,73 @@ function createActions(store) {
           getHousesWithLocation: RequestStatus.Error
         });
       }
+    },
+
+    async getAreasWithLocation(state) {
+      store.setState({
+        getAreasWithLocation: RequestStatus.Getting
+      });
+      try {
+        const areasWithLocation = await state.httpClient.get('/api/v1/area');
+        store.setState({
+          areasWithLocation,
+          getAreasWithLocation: RequestStatus.Success
+        });
+      } catch (e) {
+        store.setState({
+          getAreasWithLocation: RequestStatus.Error
+        });
+      }
+    },
+    onTitleChange(state, event) {
+      store.setState({
+        title: event.target.value
+      });
+    },
+    onLatitudeChange(state, event) {
+      store.setState({
+        latitude: event.target.value
+      });
+    },
+    onLongitudeChange(state, event) {
+      store.setState({
+        longitude: event.target.value
+      });
+    },
+    onRadiusChange(state, event) {
+      store.setState({
+        radius: event.target.value
+      });
+    },
+    async createAreasWithLocation(state, e) {
+      if (e) {
+        e.preventDefault();
+      }
+      store.setState({
+        createAreasWithLocation: RequestStatus.Getting,
+        title: '',
+        latitude: '',
+        longitude: '',
+        radius: ''
+      });
+
+      try {
+        const areasWithLocation = await state.httpClient.post('/api/v1/area', {
+          name: state.title,
+          latitude: state.latitude,
+          longitude: state.longitude,
+          radius: state.radius,
+          color: '#00000'
+        });
+        store.setState({
+          areasWithLocation,
+          createAreasWithLocation: RequestStatus.Success
+        });
+      } catch (e) {
+        store.setState({
+          createAreasWithLocation: RequestStatus.Error
+        });
+      }
```

Comment:
I don't see where it's used in this PR? Old code?

Reply:
Sorry for that my branch was an old test where I wanted to use leaflet editor toolbar to create areas. 
I separated the feature in another branch and doing this i screwed up.
=> I will correct